packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -198,6 +198,15 @@ contract OPContractsManager is ISemver {
     /// @param upgrader Address that initiated the upgrade
     event Upgraded(uint256 indexed l2ChainId, ISystemConfig indexed systemConfig, address indexed upgrader);
 
+    /// @notice Emitted when a new game type is added to a chain
+    /// @param l2ChainId Chain ID of the chain
+    /// @param gameType Type of the game being
+    /// @param newDisputeGame Address of the deployed dispute game
+    /// @param oldDisputeGame Address of the old dispute game
+    event GameTypeAdded(
+        uint256 indexed l2ChainId, GameType indexed gameType, IDisputeGame newDisputeGame, IDisputeGame oldDisputeGame
+    );
```

Comment:
this event signature was approved by proofs I've update the event based on [proofs input](https://discord.com/channels/1244729134312198194/1337143846143791205/1338880334384861195).